### DISCUSSION TITLE
PandaAi

### DISCUSSION THREAD

**Discussion by Aakidul** [microsoft/AI-For-Beginners#297] [Show and tell]
I JUST CREATED A AI MODEL NAMED PandaAi 
YOU CAN CHECK CLICKING THIS URL👇
https://github.com/Aakidul/PandaAi

THIS IS SUPER LITE WEIGHT AI MODEL IT'S SIZE AROUND 1MB
IT CAN RUN ANY DEVICE WITHOUT ANY ERROR
AND ALSO IT DOES NOT USE ANY SPECIAL DEEP LEARNING MODULES 
IT IS PURE PYTHON MODEL🤠

**Comment by ellielo** [comment]
Starting with ideas then jumping into code works well when followed by actual use cases. Tutorials on their own fall short - real understanding grows after making tiny AI tools. When others see what you build, comments come in plus attention builds up slowly. Text written with assistance from artificial intelligence often feels flat; that is where [https://cleverhumanizer.ai/](https://cleverhumanizer.ai/) steps in to add a human rhythm back in. Simple layout plus past records help when redoing work. When people talk well, skills grow faster.
